README.md

```diff
@@ -443,7 +444,7 @@ Will add a content hash to the JS and CSS filenames, generating a new filename i
 
   Type: `boolean`
 
-  Default is set to false, when set to true this will bundle a versioned css file e.g `'filename-[version].css'`.
+  Default is set to `false`, when set to true this will bundle a versioned css file e.g `'filename-[version].css'`.
```

Comment:
Looks like this one is also missing the default option..

Reply:
That's my bad, I can fix this in my PR as part of the original ticket unless @xander-marjoram wants to amend it here? :)

Reply:
@kevinrodrigues I'll fix it here, if you haven't already 👍